src/App/App.tsx

```diff
@@ -197,6 +205,25 @@ App.propTypes = {
 
 export default App;
 
+const useNrfdlLogs = () => {
+    useEffect(() => {
+        const taskId = startLogEvents(
+            getDeviceLibContext(),
+            (err?: Error) => {
+                if (err)
+                    logger.logError(
+                        'Error while listening to log messages from nrf-device-lib',
+                        err
+                    );
+            },
+            event => logNrfdlLogs(event)
```

Comment:
[Very minor] This can be simplified to:

```suggestion
            logNrfdlLogs
```